adls-plugins/docs/ADLSBatchSink-batchsink.md

```diff
@@ -10,6 +10,19 @@ Properties
 
 **path** Path to directory to store output files. The path must start with `adl://`
 
+*NOTE* 
+Either of configs {keyVaultUrl,kvKeyNames} OR {clientId,refreshTokenURL,credentials} MUST be specified.
+If former information is available, later configs are NOT required. Infact, former one(keyVault) is a more secured 
+approach to avoid specifying sensitive Service Principle details in Plugin config.
+Please refer to `Notes on using KeyVault approach` below for more details.
+
+
+**Key Vault URL** Provide Mircrosoft's KeyVault URL address from where client credentials can be fetched
+
+**Key Vault Secret Key Names** Provide key names for secret key-values to be fetched from KeyVault store. For connecting to ADLS as ClientCredentials role, a service
+principle is required which gets specified by 3 configs - clientId, clientCrdential and RefreshTokenUrl. Therefore, each
```

Comment:
I'm not sure what this means. The note above says the key vault properties are mutually exclusive from the clientid, credential, and refresh token? How are they related?